eclass/qt5-build.eclass

```diff
@@ -655,7 +655,8 @@ qt5_base_configure() {
 
 		# disable undocumented X11-related flags, override in qtgui
 		# (not shown in ./configure -help output)
-		-no-xkb -no-xrender
+		$([[ ${PV} != 5.8.9999 ]] && echo -no-xrender)
```

Comment:
This test is wrong. See how `QT5_MINOR_VERSION` and `QT5_PATCH_VERSION` are used elsewhere in the eclass.

Moreover, while it is true that that upstream commit is only present in the 5.8 branch right now, it will soon be merged into 5.9 and dev (5.9999), so you should check for `minor < 8 || (minor == 8 && patch < 1)`.